readdirp.js

```diff
@@ -199,7 +204,14 @@ function readdir(opts, callback1, callback2) {
   }
 
   function readdirRec(currentDir, depth, callCurrentDirProcessed) {
+    var args = Array.prototype.slice.call(arguments)
```

Comment:
And it may not be necessary to `slice` given how you're using it. I believe `.apply()` will work with array-like objects.